.gitignore

```diff
@@ -14,6 +14,3 @@ npm-debug.log
 /src/component-index.js
 
 .DS_Store
-
-# https://github.com/vector-im/riot-web/issues/7083
-package-lock.json
```

Comment:
something I encountered while breaking my local environment was that a `yarn-error.log` can be created. We should ignore that at all 3 levels.